cfme/tests/cloud_infra_common/test_genealogy.py

```diff
@@ -97,7 +124,7 @@ def test_compare_button_enabled():
     Bugzilla:
         1694712
     """
-    pass
+    assert create_vm_with_clone[0].genealogy.compare(*create_vm_with_clone).is_displayed
```

Comment:
In addition to checking the view is displayed, per the `testSteps` and test title, we should also check that the compare button enabled. `disabled` is a property of the `Button` class, so you should be able to use that here. 

https://github.com/RedHatQE/widgetastic.patternfly/blob/160140617f8418f9dcade587ee3dd60f6c405353/src/widgetastic_patternfly/__init__.py#L166